config/config.go

```diff
@@ -49,16 +49,18 @@ type SSLConfig struct {
 }
 
 type Config struct {
-	Hazelcast hazelcast.Config
-	SSL       SSLConfig
+	Hazelcast             hazelcast.Config
+	SSL                   SSLConfig
+	DisableAutocompletion bool
```

Comment:
How about `NoAutocompletion`?

Reply:
Sure, also renamed the flag to `no-completion` https://github.com/hazelcast/hazelcast-commandline-client/pull/102/commits/b357d82b69f70aa42dcea4c6664b450ada4262f1